extensions/objects/templates/svg-filename-editor.component.html

```diff
@@ -10,7 +10,33 @@
     <div class="fabric-canvas" ng-attr-id="<[$ctrl.canvasContainerId]>">
       <canvas ng-attr-id="<[$ctrl.canvasID]>" class="fabric-canvas-container"></canvas>
     </div>
-    <div ng-if="$ctrl.isDrawModePieChart()" class="fabric-canvas pie-chart-form">
+    <div ng-if="$ctrl.isDrawModeSvgUpload()" class="fabric-canvas custom-tool-form">
+      <div class="container upload-warning">
+        Note: The uploaded svg may not be rendered exactly.
+      </div>
+      <div ng-if="!$ctrl.isFileUploaded()">
+        <image-uploader on-file-changed="$ctrl.onFileChanged" allowed-image-formats="$ctrl.imageType">
```

Comment:
Similar to my earlier comment, naming seems off? One is plural.

Reply:
That is because only one image type is allowed in here i.e the svg.

Reply:
The naming is still off and should be fixed -- ctrl.imageType should be a list with one element and should be changed to ctrl.allowedImageFormats perhaps.

Reply:
Done.